drivers/storage/portworx/portworx.go

```diff
@@ -395,10 +396,45 @@ func (p *portworx) getCurrentMaxStorageNodesPerZone(
 	return 0, fmt.Errorf("storage disabled")
 }
 
+func isVsphere(cluster *corev1.StorageCluster) bool {
```

Comment:
Can we move this to portworx/util.go? We have many similar helper functions in there.